Sources/Swiftly/Uninstall.swift

```diff
@@ -100,10 +100,9 @@ struct Uninstall: SwiftlyCommand {
                     ?? config.listInstalledToolchains(selector: .latest).filter({ !toolchains.contains($0) }).max()
                     ?? config.installedToolchains.filter({ !toolchains.contains($0) }).max()
                 {
-                    try await Use.execute(toUse, &config)
+                    try await Use.execute(toUse, globalDefault: true, &config)
                 } else {
                     // If there are no more toolchains installed, just unuse the currently active toolchain.
-                    try Swiftly.currentPlatform.unUse(currentToolchain: toolchain)
```

Comment:
Do we still want to `unuse` the toolchain if there are no more toolchains to be set active, in order to clean up all the symlinks?

Reply:
With this change the proxy symlinks are created at install time of swiftly and remain there. Keeping them stable helps with certain shells that aggressively cache commands on the PATH, such as zsh. If there are no toolchains installed and someone tries to run "swift", "clang", or some other toolchain command then swiftly will give an error indicating that they can install one.

Reply:
interesting, so by installing swiftly then you wouldn't really be able to invoke your original swift toolchain installation?

Say I had a toolchain installed, then I decide to install swiftly, I download some toolchains with swiftly, then ultimately uninstall them all. I then go to invoke swift again under the assumption my default toolchain which was installed prior to swiftly would take precedence. Sounds like in this case that would not work because all the symlinks would still be proxying through swiftly which wouldn't be able to find a toolchain?

Reply:
Right, by installing swiftly you're giving it the ability to control which toolchain you are invoking. Note that since swiftly maintains its own record of toolchains it installed, any that you've installed previously wouldn't be known to swiftly since they could be in virtually any location on the system.

The user can take control temporarily by overriding their PATH to the external toolchain that they want to use, or uninstall swiftly.